todo_app/app.py

```diff
@@ -6,43 +8,45 @@
     url_for,
 )
 
-from todo_app.flask_config import Config
-from todo_app.data.session_items import (
-    get_items, 
-    add_item,
-    get_item,
-    save_item,
-    delete_item,
-)
+from todo_app.config import Config, TrelloConfig
+from todo_app.data.trello import Trello, CardStatus
```

Comment:
Now that you've removed the session_items file (and therefore aren't storing session state), we could also remove the `SECRET_KEY` variable (and by proxy, the `Config` class)